economy.md

```diff
@@ -2,22 +2,16 @@
 
 ## bal
 
-- Quality.
-
-  <img align="top" src="https://user-images.githubusercontent.com/86816930/142962190-9722fd2a-2b7e-4851-a03e-f182e59f3bfc.png">
+- Test the fucking command and you'll find out glory. (you apparentally are incapable of doing this whenever coding!)
 
 ## buy
 
 - Doesn't respoond if you specify an invalid item ID.
-- Formatting of literally every part of this command is horrible.
+- "daily" and weekly" is definitely a clear interface of what an ID could be
```

Comment:
```suggestion
```

There is nothing on this command that says anything about an ID, nor is it unclear (possibly the only part of Lacan with a clear UI).